src/main/java/org/spring/FullVersion/Human.java

```diff
@@ -23,18 +21,20 @@ public class Human {
     private int iq;
     private Family family;
     private Map<String, String> schedule;
+    private String gender;
```

Comment:
no handle it in another way, checking instance of Man or Woman inside prettyFormat